pkg/rtc/participant.go

```diff
@@ -1076,9 +1076,10 @@ func (p *ParticipantImpl) onSubscriberOffer(offer webrtc.SessionDescription) {
 	})
 	if err != nil {
 		prometheus.ServiceOperationCounter.WithLabelValues("offer", "error", "write_message").Add(1)
-	} else {
-		prometheus.ServiceOperationCounter.WithLabelValues("offer", "success", "").Add(1)
+		p.params.Logger.Errorw("could not send offer", err)
```

Comment:
I see already have log message type in `Participant.writeMessage`, is here required?

Reply:
Good point. Don't need this. Will remove.

Reply:
Removed in this commit - https://github.com/livekit/livekit/pull/933/commits/287de8aa653e8b6830957186cc19e53560410046